### DISCUSSION TITLE
Strange letter spacing in codespaces terminal

### DISCUSSION THREAD

**Discussion by whatsinfinitum** [community/community#41052] [Codespaces]
### Select Topic Area

Bug

### Body

The terminal in my codespaces (in my personal account) has strange letter spacing. (this doesn't happen on my github staff account). 

<img width="1158" alt="Screenshot 2022-12-08 at 11 29 35" src="https://user-images.githubusercontent.com/99569094/206341792-52ee298a-f9ea-4840-ae7f-251825e5da00.png">

**Comment by jfullstackdev** [comment]
that's not happening in my free account, I just loaded it by now 

![image](https://user-images.githubusercontent.com/47092464/206383548-feb358a0-aeeb-4e7b-a2c4-593db285fff8.png)

did you change the settings ? or maybe some VS Code extension ? GitHub extensions ?

**Comment by geekceo** [comment]
@whatsinfinitum I have same problem when I installed old japan visual novel and turn on locale changer to make it english language. May be you have same situation, check system locales. Or it may be VS Code custom fonts

**Comment by whatsinfinitum** [comment]
thanks I'll try that and report back.
